synthetic_data/generate_readme.sh

```diff
@@ -0,0 +1,54 @@
+export s3_path=https://instruct-lab.s3.us-east.cloud-object-storage.appdomain.cloud
```

Comment:
Please do not add the synthetic data folder back!!

Reply:
I'm not sure why my PRs keep grabbing that commit! Fixing!